src/model/backend/ftp.go

```diff
@@ -96,7 +104,7 @@ func (f Ftp) LoginForm() Form {
 				Name:        "advanced",
 				Type:        "enable",
 				Placeholder: "Advanced",
-				Target:      []string{"ftp_path", "ftp_port", "ftp_conn"},
+				Target:      []string{"ftp_path", "ftp_port", "ftp_conn", "ftp_tls"},
```

Comment:
The `"ftp_tls"` doesn't relates to anything in this context

Reply:
`ftp_tls` is removed